scripts/install-travis-script.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+set -o nounset
+set -o xtrace
+
+if [ "$RUN_PROD" = true ]; then
+  npm install -g bower
+  npm install --production
+  bower update
```

Comment:
Is there a reason we don't also use bower install here?

Reply:
Nope, just missed it. Thanks